src/CLI/commands/build.ts

```diff
@@ -138,16 +139,19 @@ export = ts.identity<yargs.CommandModule<object, BuildFlags & Partial<ProjectOpt
 				setupProjectWatchProgram(data, projectOptions.usePolling);
 			} else {
 				const program = createProjectProgram(data);
+				const compilerOptions = program.getCompilerOptions();
 				const pathTranslator = createPathTranslator(program, data);
 				cleanup(pathTranslator);
 				copyInclude(data);
-				copyFiles(data, pathTranslator, new Set(getRootDirs(program.getCompilerOptions())));
-				const emitResult = compileFiles(
-					program.getProgram(),
-					data,
-					pathTranslator,
-					getChangedSourceFiles(program),
-				);
+				copyFiles(data, pathTranslator, new Set(getRootDirs(compilerOptions)));
+
+				const changedSourceFiles = getChangedSourceFiles(program);
+				const changedCompilableFiles = changedSourceFiles.filter(v => !v.isDeclarationFile);
```

Comment:
Since this isn't used in the declaration code path, it's slightly easier to read if it goes below the if block
(Same goes for this block in setupProjectWatchProgram)